apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -297,15 +297,18 @@ type Image struct {
 	URL string `json:"url"`
 
 	// Checksum is the checksum for the image.
-	Checksum string `json:"checksum"`
+	Checksum string `json:"checksum,omitempty"`
 
 	// ChecksumType is the checksum algorithm for the image.
 	// e.g md5, sha256, sha512
 	ChecksumType ChecksumType `json:"checksumType,omitempty"`
 
 	// DiskFormat contains the format of the image (raw, qcow2, ...)
 	// Needs to be set to raw for raw images streaming
-	// +kubebuilder:validation:Enum=raw;qcow2;vdi;vmdk
+	// Note live-iso means an iso referenced by the url will be live-booted
+	// and not deployed to disk, and in this case the checksum options
+	// should be omitted.
```

Comment:
How about instead of "should be omitted" saying "will be ignored"?

It would also be nice to clean up the rendered version a bit by adding `.` to the end of the previous 2 sentences.

Reply:
Done!